charts/chainlink-cluster/dashboard/dashboard.go

```diff
@@ -58,16 +225,14 @@ func (m *CLClusterDashboard) Opts() []dashboard.Option {
 }
 
 // logsRowOption returns a row option for a node's logs with name and instance selector
-func (m *CLClusterDashboard) logsRowOption(name, instanceSelector string) row.Option {
+func (m *CLClusterDashboard) logsRowOption(name, q string) row.Option {
```

Comment:
I wonder if here we could provide some query builder and then use a more specific type than `string` to avoid syntax errors?

Reply:
Sadly enough it'd require changing all declarations of metrics to apply the type system.
An easier approach is to use linter in the next PRs since even if your requests are statically correct that doesn't guarantee they'd work, so linting can solve some problems with a small overhead and it'd be simple for devs.